internal/scheduler/tasks_test.go

```diff
@@ -41,7 +41,7 @@ func TestExecuteTask(t *testing.T) {
 	assert.Error(t, et("Download", []string{"foo"}), "Invalid json")
 	assert.EqualError(t, et("Download", []string{`{"workersnum": 0, "fileurls": [] }`}),
 		"Files to download are not specified", "Download with empty files should fail")
-	assert.Error(t, et("Download", []string{`{"workersnum": 0, "fileurls": ["http://foo.bar"], "destpath": "" }`}),
+	assert.Error(t, et("Download", []string{`{"workersnum": 0, "fileurls": ["pgtt_http://foo.bar"], "destpath": "" }`}),
```

Comment:
```suggestion
	assert.Error(t, et("Download", []string{`{"workersnum": 0, "fileurls": ["http://foo.bar"], "destpath": "" }`}),
```

Reply:
Sorry, mistake append. This file should not be changed.